### DISCUSSION TITLE
How to add and change fader address  using MCU::VOLUME_ more than 8

### DISCUSSION THREAD

**Discussion by pukulsesuatu** [tttapa/Control-Surface#784] [Q&A] [answered]
Hi everyone, I am using 8 motorize faders by using MCU::VOLUME_1 to MCU::VOLUME_8. I want to change the fader address with a bankable button, but I see that MCU::VOLUME is only up to 8. how can I change the fader volume to more than 8. thanks, hope someone can answer it.

**Comment by Veroledez** [comment, marked as answer]
Hola, el protocolo mackie  solo tiene esas definiciones de volume_1 a volume_8,  y volume_9 creo que esta asignado al master, volume_10  en adelante no existe, los bancos ya los maneja el script midi de tu daw, para cambiar de 8 en 8 esta BANK_LEFT. y. BANK_RIGHT  usalos con botones, es el daw el que maneja los bancos en este caso y se encarga de todo

**Reply by pukulsesuatu** [reply]
Thanks alot, I am trying using BANK_RIGHT and BANK_LEFT and works, thanks.

**Comment by tttapa** [comment]
The original Mackie Control Universal only has 8 channel faders (+master volume). If you want more, you'll need to use MCU banks (using `BANK_LEFT`/`BANK_RIGHT`, as mentioned by Veroledez, this is different from Control Surface's `Bank` class), or use multiple virtual MIDI cables that you map as Mackie Control Universal eXTenders.

Alternatively, you can check if your DAW supports a different protocol than MCU, because MCU is pretty limited.

**Reply by pukulsesuatu** [reply]
Its works I am use `BANK_LEFT`/`BANK_RIGHT` with custom button  [Custom-MIDI-Output-Element.ino](https://tttapa.github.io/Control-Surface-doc/Doxygen/d4/d3b/Custom-MIDI-Output-Element_8ino-example.html#_a11) and change address to `Btn bank_Right{
  5,
  {MCU::BANK_RIGHT},
  0x7F,
};`   thanks Pieter for the information and MIDI Control Surface library that cool library
